README.md

```diff
@@ -18,7 +18,7 @@
 This repository contains verified dlt sources and demo pipelines for each source that you can use as a starting point for your project. 
 
 ## How to add a verified source to your dlt project
-`dlt` offers an `init` command that will clone and inject any source with an example pipeline from this repository into your project, setup the credentials and python dependencies. Please follow the step by step instructions in our [docs](https://dlthub.com/docs/walkthroughs/add-a-pipeline). 
+`dlt` offers an `init` command that will clone and inject any source with an example pipeline from this repository into your project, setup the credentials and python dependencies. Please follow the step by step instructions in our [docs](https://dlthub.com/docs/walkthroughs/create-a-pipeline).
```

Comment:
make sense to use https://dlthub.com/docs/walkthroughs/add-a-verified-source page

Reply:
Oh, even better! Thanks!